src/olympia/amo/tests/test_models.py

```diff
@@ -68,34 +68,34 @@ def test_multiple_ignored(self):
 
     def test_change_called_on_new_instance_save(self):
         for create_addon in (Addon, Addon.objects.create):
-            addon = create_addon(public_stats=False, type=amo.ADDON_EXTENSION)
-            addon.public_stats = True
+            addon = create_addon(view_source=False, type=amo.ADDON_EXTENSION)
+            addon.view_source = True
```

Comment:
can we fix this in a way that doesn't rely on `.view_source` - it's supposed to be removed from the model too in https://github.com/mozilla/addons-server/issues/12842

Reply:
#12842 is a contrib:welcome issue too, fwiw, so it'll be making that more difficult for contributors to fix.

Reply:
ah yes, sure. I didnt know about this. Thanks!